src/banana/toy.py

```diff
@@ -9,62 +9,6 @@
 import atexit
```

Comment:
This one you can even remove at this point.

Or at least add an option to silence like:
```py
verbosity = True
def SetVerbosity(level):
    if level == 0:
        verbosity = False
        
def ciao():
    if not verbosity:
        return
    ...
```